app/Http/Controllers/UserController.php

```diff
@@ -125,4 +125,339 @@ public function destroy(User $user)
+    {
+        $user1 = factory(User::class)->create(['role' => UserType::USER]);
+        $this->actingAs($user1);
+        $user = factory(User::class)->create(
+            [
+                'name' => "Name Create",
+                'email' => 'email@gmail.com',
+                'password' => '12345678',
+                'role' => '1',
+                'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+            ]
+        );
+        $params = [
+            'name' => "Name Update",
+            'email' => 'emaiupdatel@gmail.com',
+            'password' => '12345678',
+            'role' => '1',
+            'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+        ];
+        $response = $this->put(route('users.update', $user->id), $params);
+        $response->assertStatus(302);
+    }
+
+    public function testUpdateUserRequireName()
+    {
+        $admin = factory(User::class)->create(['role' => UserType::ADMIN]);
+        $this->actingAs($admin);
+        $user = factory(User::class)->create(
+            [
+                'name' => "Name Create",
+                'email' => 'email@gmail.com',
+                'password' => '12345678',
+                'role' => '1',
+                'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+            ]
+        );
+        $params = [
+            'name' => null,
+            'email' => 'emaiupdatel@gmail.com',
+            'password' => '12345678',
+            'role' => '1',
+            'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+        ];
+        $response = $this->put(route('users.update', $user->id), $params);
+        $response
+            ->assertStatus(302)
+            ->assertSessionHasErrors('name');
+    }
+
+    public function testUpdateUserNameMinLength()
+    {
+        $admin = factory(User::class)->create(['role' => UserType::ADMIN]);
+        $this->actingAs($admin);
+        $user = factory(User::class)->create(
+            [
+                'name' => "Name Create",
+                'email' => 'email@gmail.com',
+                'password' => '12345678',
+                'role' => '1',
+                'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+            ]
+        );
+        $params = [
+            'name' => '123',
+            'email' => 'emaiupdatel@gmail.com',
+            'password' => '12345678',
+            'role' => '1',
+            'avatar' => UploadedFile::fake()->image('avatar.jpg'),
+        ];
+        $response = $this->put(route('users.update', $user->id), $params);
+        $response
+            ->assertStatus(302)
+            ->assertSessionHasErrors('name');
+    }
+
+    public function testUpdateUserNameMaximumLength()
+    {
+        $admin = factory(User::class)->create(['role' => UserType::ADMIN]);
+        $this->actingAs($admin);
+        $user = factory(User::class)->create(
+            [
+                'name' => "Name Create",
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 261</br>String "Name Create" does not require double quotes; use single quotes instead